h2o-r/h2o-package/R/models.R

```diff
@@ -3189,6 +3192,56 @@ h2o.getModelTree <- function(model, tree_number, tree_class = NA) {
     tree@tree_class <- res$tree_class
   }
   
+  if(is.logical(res$levels)){ # Vector of NAs is recognized as logical type in R
```

Comment:
Levels are a two-dim array. If there are no arrays at all, an array of nulls is sent. This is translated to a vector of NAs. Class of such vector is `logical` and would not fit our purpose. Therefore, such vector of NAs is translated manually into a list of NULLs.